pkg/component/component.go

```diff
@@ -294,33 +328,39 @@ func GetComponentSource(client *occlient.Client, componentName string, applicati
 		return "", "", fmt.Errorf("unsupported component source type %s", sourceType)
 	}
 
-	log.Debugf("Source for component %s is %s (%s)", componentName, sourcePath, sourceType)
+	log.Debugf("Source for component %s is %s (%s)", namespacedOpenShiftObject, sourcePath, sourceType)
```

Comment:
text in debug message mentions component, so the first parameters should be the  component name not name of the openshift object